.github/workflows/build-and-test-freebsd.yml

```diff
@@ -0,0 +1,52 @@
+name: Build and Test (FreeBSD)
+
+on:
+  pull_request:
+  push:
+    branches:
+      - 'main'
+      - 'devel/**'
+  workflow_dispatch:
+
+jobs:
+  build:
+    if: github.event.pull_request.draft == false
+    name: Build
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Get number of CPU cores
+      uses: SimenB/github-actions-cpu-cores@v2
+      id: cpu-cores
+
+    - name: Checkout Code
+      uses: actions/checkout@v4
+
+    - name: Checkout Test Files
+      uses: actions/checkout@v4
+      with:
+        repository: neomutt/neomutt-test-files
+        path: test-files
+
+    - name: Build and Test on FreeBSD
+      uses: vmactions/freebsd-vm@v1
+      with:
+        usesh: true
+        prepare: |
+          pkg install -y perl5 pkgconf gettext libxslt docbook-xsl db5 gdbm gpgme gnutls kyotocabinet tokyocabinet lmdb lua54 pcre2 notmuch qdbm rocksdb cyrus-sasl tdb zstd
+        run: |
+          cd test-files
+          ./setup.sh
+          cd -
+          mkdir build
+          cd build
```

Comment:
Any reason for building out-of-tree?
[`build-and-test.yml`](https://github.com/neomutt/neomutt/blob/main/.github/workflows/build-and-test.yml) already checks this.

Reply:
no, I can skip that